extensions/indexes/lucene/src/org/exist/indexing/lucene/LuceneIndexWorker.java

```diff
@@ -487,6 +488,13 @@ protected void setOptions(Properties options, CommonQueryParserConfiguration par
             else
                 parser.setMultiTermRewriteMethod(MultiTermQuery.CONSTANT_SCORE_BOOLEAN_QUERY_REWRITE);
         }
+        option = options.getProperty(OPTION_LOWERCASE_EXPANDED_TERMS);
+        if (option != null) {
+            if (option.equalsIgnoreCase("yes"))
```

Comment:
@dizzzz you just confuse @claudius108  -)

@claudius108 at code you provide `if` make sense because it set none-boolean values, but at